packages/core/src/operations/ensure.ts

```diff
@@ -0,0 +1,9 @@
+import { Operation } from '../operation';
+
+export function *ensure<T>(operation?: Operation<T>): Operation<void> {
```

Comment:
Should we allow undef here? Seems like a guaranteed hang?

Reply:
True. Seems a bit useless to allow it.